e2e/runner/setup_solana.go

```diff
@@ -87,6 +87,10 @@ func (r *E2ERunner) SetupSolana(deployerPrivateKey string) {
 
 	err = r.ensureSolanaChainParams()
 	require.NoError(r, err)
+
+	// deploy test spl
+	tokenAccount := r.DeploySPL(&privkey, true, uint64(1_000_000))
```

Comment:
What does 1 million mean in this context?

Reply:
For code readability, you can do
```go

const initialTokensMint = 1_000_000

tokenAccount := r.DeploySPL(&privkey, true, initialTokensMint)

// ...
```

Reply:
Unless it's used in multiple places, I find the const to be less readable.